### PR TITLE
[FR] Update `Community\Project_Loved`, `How_you_can_help!`, `Community_Mentorship_Program`, Add `Community`, `Forum`, `Feature_Requests`, `Help`, `Modding_Queues`, `Off-Topic`, `Video_series`

### PR DESCRIPTION
**Pull request by DoryanR**
Translation of the following wiki pages:

- `Video_series`
- `Community\Forum\Feature_Requests`
- `Community\Forum`
- `Community\Forum\Help` 
- `Community\Forum\Modding_Queues`
- `Community\Forum\Off-Topic`
- `Community\Video_series`

Update of the translations of the following wiki pages:

- `Community\Community_Mentorship_Program`
- `Community\Project_Loved`
- `Community\How_you_can_help!`
- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] Depends on #6523 for `Community\Forum`

*Feel free to merge if the delay is too long for a review by a native.*

### CODE REVIEW

File: wiki/Community/fr.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Communauté
+
+Cette page répertorie les moyens par lesquels les personnes de tout le site d'osu! peuvent entrer en contact les unes avec les autres. Si vous êtes à la recherche d'opportunités de contribution, mais ne savez pas par où commencer, assurez-vous de lire [Comment vous pouvez aider !](/wiki/Community/How_you_can_help!).
+
+## Places
+
+- [Internet Relay Chat](/wiki/Community/Internet_Relay_Chat) (IRC)
```
Comment (DoryanR) [inline wiki/Community/fr.md:7]:
```suggestion
## Places

- [Forum](/wiki/Community/Forum)
- [Internet Relay Chat](/wiki/Community/Internet_Relay_Chat) (IRC)
```

File: wiki/Community/Forum/Help/fr.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+stub: true
+---
+
+# Help
+
+*Voir également : [Centre d'aide](/wiki/Help_Centre)*
+
+[**Help**](https://osu.ppy.sh/community/forums/5) est un forum où les joueurs peuvent recevoir une assistance technique de la part des membres de l'[Équipe du support](/wiki/People/The_Team/Support_Team) ainsi que des utilisateurs réguliers.
```
Comment (Ganondorf-1) [inline wiki/Community/Forum/Help/fr.md:9]:
```suggestion
[**Help**](https://osu.ppy.sh/community/forums/5) est un forum où les joueurs peuvent recevoir une assistance technique de la part des membres de l'[Équipe du support](/wiki/People/The_Team/Support_Team) ou des autres utilisateurs.
```

File: wiki/Community/Forum/Modding_Queues/fr.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+stub: true
+---
+
+# Modding Queues
+
+[**Modding Queues**](https://osu.ppy.sh/community/forums/60) est un forum dédié aux demandes de [review des beatmaps](/wiki/Modding), ou mods. Il est constitué de fils de discussion, ou *queues*, appartenant à différents moddeurs et mappeurs. Chaque file d'attente a des exigences et des limites individuelles pour l'obtention des mods, qui doivent être respectées par les visiteurs afin de recevoir un mod.
```
Comment (Ganondorf-1) [inline wiki/Community/Forum/Modding_Queues/fr.md:7]:
```suggestion
[**Modding Queues**](https://osu.ppy.sh/community/forums/60) est un forum dédié aux demandes de [review des beatmaps](/wiki/Modding), ou de modding. Il est constitué de fils de discussion, ou *queues*, appartenant à différents moddeurs et mappeurs. Chaque file d'attente a des exigences et des limites individuelles pour l'obtention des mods, qui doivent être respectées par les visiteurs afin de recevoir un mod.
```

File: wiki/Community/Forum/fr.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+## Structure
+
+Chacun des forums d'osu! contient une variété de **sujets**, parfois appelés fils de discussion, ainsi que des catégories plus petites, ou **sous-forums**. La plupart des forums sont uniquement en anglais, mais il existe également un certain nombre de forums [spécifiques à une langue](https://osu.ppy.sh/community/forums#forum-23) pour les communautés locales suffisamment importantes.
+
+Bien que certains forums, tels que [Mapping Discussion](https://osu.ppy.sh/community/forums/56), aient des exigences ou des directives spécifiques, tous suivent les [règles de la communauté](/wiki/Rules).
+
+## Rôles et groupes d'utilisateurs
+
+Certains utilisateurs peuvent appartenir à un ou plusieurs groupes personnalisés, en fonction de leurs tâches communautaires, de leurs droits d'accès ou de leur activité passée. Une liste complète des groupes d'utilisateurs se trouve à l'adresse [L'équipe § Groupes d'utilisateurs](/wiki/People/The_Team#groupes-d'utilisateurs).
```
Comment (Ganondorf-1) [inline wiki/Community/Forum/fr.md:24]:
```suggestion
Certains utilisateurs peuvent appartenir à un ou plusieurs groupes personnalisés, en fonction de leurs tâches communautaires, de leurs droits d'accès ou de leur activité passée. Une liste complète des groupes d'utilisateurs se trouve sur la page [L'équipe § Groupes d'utilisateurs](/wiki/People/The_Team#groupes-d'utilisateurs).
```

File: wiki/Community/Forum/fr.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+En plus des badges de groupes d'utilisateurs fournis par l'appartenance à un groupe, les utilisateurs peuvent avoir des [titres d'utilisateur personnalisés](/wiki/User_title) affichés sous l'avatar dans chacun de leurs messages.
+
+### Modération
+
+*Voir également : [Reporting bad behaviour § Forum](/wiki/Reporting_Bad_Behaviour#forum)*
```
Comment (Ganondorf-1) [inline wiki/Community/Forum/fr.md:30]:
```suggestion
*Voir également : [Signaler un mauvais comportement § Forum](/wiki/Reporting_Bad_Behaviour#forum)*
```

File: wiki/Community/Forum/fr.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+Les [modérateurs](/wiki/People/The_Team/Global_Moderation_Team) sont des utilisateurs qui veillent à ce que les forums restent propres et appropriés, et répondent aux questions à ce sujet. Les modérateurs peuvent modifier ou supprimer des messages ou des sujets inappropriés, épingler des sujets et les déplacer entre différents forums, et empêcher temporairement les contrevenants de poster en les [réduisant au silence](/wiki/Silence).
+
+## Forums spéciaux
+
+Certains forums ont des objectifs différents qui ne se limitent pas à la discussion. Par conséquent, ils ont des règles et/ou des capacités techniques supplémentaires.
+
+- [Help](/wiki/Community/Forum/Help) : l'assistance technique de l'[Équipe du support](/wiki/People/The_Team/Support_Team) et des utilisateurs réguliers.
```
Comment (Ganondorf-1) [inline wiki/Community/Forum/fr.md:38]:
```suggestion
- [Help](/wiki/Community/Forum/Help) : l'assistance technique de l'[Équipe du support](/wiki/People/The_Team/Support_Team) et autres utilisateurs.
```

File: wiki/Community/How_you_can_help!/fr.md [hunk 7]
```diff
@@ -1,74 +1,74 @@
...
 
-Afin de maintenir les canaux de discussion officiels de la communauté sains, osu! a une équipe de modération dédiée qui modère continuellement les forums et les chats in-game. 
+Afin de garantir la sécurité des centres officiels de la communauté, osu! dispose d'une équipe de modération spécialisée qui surveille les forums et les tchats en ligne 24 heures sur 24. 
 
-Heureusement, il est facile d'aider l'équipe de modération à garder ces endroits sûrs. Voici unes des nombreuses façons d'aider :
+Heureusement, il est encore facile pour vous de contribuer à la sécurité du site. Voici quelques-unes des nombreuses façons dont vous pouvez nous aider :
 
-- Suivez vos canaux favoris sur le [chat in-game d'osu!](/wiki/Chat_Console) ou sur les forums.
-- Si vous voyez quelque chose que vous pensez inappropriés ou dérangeant, utilisez [la commande `!report`](/wiki/Reporting_Bad_Behaviour) dans le chat in-game pour notifier les modérateurs.
+- Suivez vos channels préférées dans le [tchat en jeu d'osu!](/wiki/Chat_Console), ou sur les forums.
```
Comment (Ganondorf-1) [inline wiki/Community/How_you_can_help!/fr.md:19]:
```suggestion
- Suivez vos canaux de discussion préférés dans le [tchat en jeu d'osu!](/wiki/Chat_Console), ou sur les forums.
```

File: wiki/Community/How_you_can_help!/fr.md [hunk 8]
```diff
@@ -1,74 +1,74 @@
...
+- Choisissez un forum qui suscite votre intérêt :
   - [Help](https://osu.ppy.sh/community/forums/5)
   - [Mapping Discussion](https://osu.ppy.sh/community/forums/56)
   - [Storyboarding](https://osu.ppy.sh/community/forums/20)
   - [Gameplay & Rankings](https://osu.ppy.sh/community/forums/13)
-- Marquez le forum comme lu en utilisant le lien en haut a droite.
-- Suivez les nouveaux threads, lisez chacun d'eux et aidez quand vous pouvez, vous pouvez utiliser l'icone non lue pour être sûr de les garder actifs.
-- Lisez les [guides disponibles](/wiki/Guides) pour peaufiner votre connaissance d'osu!. Si vous pensez que certains guides ne sont pas à jour ou mérite d'être traduits, vous pouvez y contribuer !
+- Marquez le forum comme lu en utilisant le lien en bas.
+- Suivez tous les nouveaux fils de discussion, lisez chacun d'entre eux et apportez votre aide quand vous le pouvez ; vous pouvez utiliser les icônes de unread pour vous assurer de rester à jour.
```
Comment (Ganondorf-1) [inline wiki/Community/How_you_can_help!/fr.md:34]:
```suggestion
- Suivez tous les nouveaux fils de discussion, lisez chacun d'entre eux et apportez votre aide quand vous le pouvez ; vous pouvez utiliser les icônes "non lu" pour vous assurer de rester à jour.
```

File: wiki/Community/How_you_can_help!/fr.md [hunk 9]
```diff
@@ -1,74 +1,74 @@
...
+- Lisez les [guides disponibles](/wiki/Guides) pour parfaire vos connaissances d'osu!. Si les guides qui s'y trouvent ne vous satisfont pas, vous pouvez essayer d'y contribuer !
 
-## Aider à vérifier/mod des beatmaps!
+## Aidez-nous à vérifier/modder les beatmaps !
 
-osu! a une équipe d'évaluateurs dédiée pour vérifier si les [beatmaps](/wiki/Beatmap) sont prêtes a être ranked et jouée par tous. Cependant, cela ne veux pas dire que vous ne pouvez pas y contribuer :
+osu! dispose d'une équipe de reviewers qui vérifient si les [beatmaps](/wiki/Beatmap) sont prêts à être classés et joués par tous. Cependant, cela ne signifie pas que vous ne pouvez pas contribuer à ce secteur :
```
Comment (Ganondorf-1) [inline wiki/Community/How_you_can_help!/fr.md:39]:
```suggestion
osu! dispose d'une équipe de reviewers qui vérifient si les [beatmaps](/wiki/Beatmap) sont prêtes à être classées et jouées par tous. Cependant, cela ne signifie pas que vous ne pouvez pas contribuer à ce secteur :
```

File: wiki/Community/How_you_can_help!/fr.md [hunk 10]
```diff
@@ -1,74 +1,74 @@
...
-- Lire le [guide de modding](/wiki/Modding) et les [critères de ranking](/wiki/Ranking_Criteria) pour connaitre les bases du modding ainsi que les différents critères qui rendent une beatmap prête pour le statut "Classée" (Ranked).
-- Rejoignez les chats in-game `#mapping` et `#modreqs` et conversez ou gardez un œil sur le canal pour apporter votre aide si nécessaire.
-- Si le modding vous intéresse, créez une [Modding Queue](https://osu.ppy.sh/community/forums/60) pour permettre aux créateurs de beatmaps de pouvoir vous demander des mods.
+- Regardez ou cherchez dans la liste [Pending Beatmaps](https://osu.ppy.sh/beatmapsets?s=pending) les artistes ou les musiques que vous aimez et/ou auxquels vous voulez contribuer.
+- Jouez les beatmaps aux stades [*En attente* ou *Work in progress*](/wiki/Beatmap/Category#work-in-progress-et-en-attente) et laissez vos commentaires sur la page de discussion de la beatmap.
+- Lisez le [Guide de modding](/wiki/Modding) et les [Critères de classement](/wiki/Ranking_Criteria) pour connaître les bases du modding technique et savoir ce qui rend un beatmap classable.
```
Comment (Ganondorf-1) [inline wiki/Community/How_you_can_help!/fr.md:43]:
```suggestion
- Lisez le [Guide de modding](/wiki/Modding) et les [Critères de classement](/wiki/Ranking_Criteria) pour connaître les bases du modding technique et savoir ce qui rend une beatmap classable.
```

File: wiki/Community/How_you_can_help!/fr.md [hunk 11]
```diff
@@ -1,74 +1,74 @@
...
 
-## Contribuer au futur d'osu!
+## Contribuez à l'avenir d'osu!
 
-Démarré en 2017, le projet osu!lazer est le centre d'interêt actuel de l'équipe d'osu!, accompagné du nouveau site web, qui est celui que vous voyez actuellement ! Si vous voulez aider à améliorer ces projets, ci-dessous voici quelques exemples-type de contributions :
+Démarré en 2017, le projet osu!lazer est le centre d'intérêt actuel de [l'équipe d'osu!](/wiki/People/The_Team), ainsi que le nouveau site web, qui est ce que vous voyez aujourd'hui ! Si vous souhaitez aider à rendre ces projets encore meilleurs et nous rapprocher de l'avenir d'osu!, il y a un certain nombre de choses que vous pouvez faire, mais ci-dessous ne sont que quelques exemples :
 
-- Aidez le projet [osu!lazer](https://github.com/ppy/osu). Il y a encore beaucoup de fonctionnalités à implémenter sur la nouvelle version d'osu!, et vous pouvez nous aider !
-- Rendez le [nouveau site web](https://github.com/ppy/osu-web) meilleur.
+- Aidez le projet [osu!lazer](https://github.com/ppy/osu). Il y a tellement de choses à implémenter dans la nouvelle version d'osu! et vous pouvez aider à les réaliser !
```
Comment (Ganondorf-1) [inline wiki/Community/How_you_can_help!/fr.md:63]:
```suggestion
- Aider le projet [osu!lazer](https://github.com/ppy/osu). Il y a tellement de choses à implémenter dans la nouvelle version d'osu! et vous pouvez aider à les réaliser !
```

File: wiki/Community/Project_Loved/fr.md [hunk 12]
```diff
@@ -1,25 +1,21 @@
...
----
-
 # Project Loved
 
 Le **Project Loved** est la plateforme de vote qui promeut les beatmaps dans la catégorie [loved](/wiki/Beatmap/Category#loved). Il est géré principalement par la [Project Loved Team](/wiki/People/The_Team/Project_Loved_Team).
 
 ## Déplacer une beatmap vers Loved
 
-Les beatmaps sont déplacés vers loved par un processus de nomination et de vote.
+Les beatmaps sont déplacés vers la catégorie loved par un processus de nomination et de vote.
```
Comment (Ganondorf-1) [inline wiki/Community/Project_Loved/fr.md:7]:
```suggestion
Les beatmaps sont déplacées vers la catégorie loved par un processus de nomination et de vote.
```

File: wiki/Community/Project_Loved/fr.md [hunk 13]
```diff
@@ -1,25 +1,21 @@
...
 
 Les membres de l'équipe du Project Loved appelés **capitaines** nomment régulièrement des beatmaps pour un mode de jeu spécifique. Ces ensembles proviennent presque toujours de propositions de la communauté d'osu!.
```
Comment (Ganondorf-1) [inline wiki/Community/Project_Loved/fr.md:9]:
```suggestion
Les membres de l'équipe du Project Loved appelés **capitaines** nomment régulièrement des beatmaps pour un mode de jeu spécifique. Ces beatmapsets proviennent presque toujours de propositions de la communauté d'osu!.
```

File: wiki/Community/Project_Loved/fr.md [hunk 14]
```diff
@@ -1,25 +1,21 @@
...
 
-Pour soumettre un beatmap pour loved, veuillez visiter la [page d'envoi d'une beatmap](https://loved.sh/submit) du site web du Project Loved et remplir le formulaire. Une fois le formulaire envoyé, vous pourrez voir votre proposition dans la liste de son mode de jeu respectif avec les autres. Vous trouverez ci-dessous les listes où ces propositions peuvent être consultées :
+Pour soumettre un beatmap pour la catégorie loved, veuillez visiter la [page de soumission d'une beatmap](https://loved.sh/submit) du site web du Project Loved et remplir le formulaire. Une fois le formulaire envoyé, vous pourrez voir votre proposition dans la liste de son mode de jeu respectif avec les autres. Vous trouverez ci-dessous les listes où ces propositions peuvent être consultées :
```
Comment (Ganondorf-1) [inline wiki/Community/Project_Loved/fr.md:11]:
```suggestion
Pour soumettre une beatmap pour la catégorie loved, veuillez visiter la [page de soumission de beatmap](https://loved.sh/submit) du site web du Project Loved et remplir le formulaire. Une fois le formulaire envoyé, vous pourrez voir votre proposition dans la liste de son mode de jeu respectif avec les autres. Vous trouverez ci-dessous les listes où ces propositions peuvent être consultées :
```

File: wiki/Community/Project_Loved/fr.md [hunk 15]
```diff
@@ -1,25 +1,21 @@
...
 
 - [Proposition pour le mode osu!](https://loved.sh/submissions/osu)
 - [Proposition pour le mode osu!taiko](https://loved.sh/submissions/taiko)
 - [Proposition pour le mode osu!catch](https://loved.sh/submissions/fruits)
 - [Proposition pour le mode osu!mania](https://loved.sh/submissions/mania)
 
-Les mapsets nominés sont ensuite publiés sur le forum [Project Loved](https://osu.ppy.sh/community/forums/120), où les membres de la communauté peuvent voter s'ils souhaitent que le mapset soit loved ou non. Les maps qui atteignent un certain pourcentage de votes positifs à la fin du scrutin sont transférées dans la catégorie "Loved".
+Les beatmaps nominés sont ensuite publiés sur le forum [Project Loved](https://osu.ppy.sh/community/forums/120), où les membres de la communauté peuvent voter s'ils souhaitent que la beatmap soit "loved" ou non. Les beatmaps qui atteignent un certain pourcentage de votes positifs à la fin du scrutin sont transférées dans la catégorie Loved.
```
Comment (Ganondorf-1) [inline wiki/Community/Project_Loved/fr.md:18]:
```suggestion
Les beatmaps nominées sont ensuite publiées sur le forum [Project Loved](https://osu.ppy.sh/community/forums/120), où les membres de la communauté peuvent voter s'ils souhaitent que la beatmap soit "loved" ou non. Les beatmaps qui atteignent un certain pourcentage de votes positifs à la fin du scrutin sont transférées dans la catégorie Loved.
```

File: wiki/Community/Project_Loved/fr.md [hunk 16]
```diff
@@ -29,10 +25,10 @@ Certains critères très minimes doivent être respectés pour que les beatmaps
 - Son créateur ne désapprouve pas le fait qu'elle soit prêté.
```
Comment (Ganondorf-1) [inline wiki/Community/Project_Loved/fr.md:25]:
```suggestion
- Son créateur ne désapprouve pas le fait qu'elle soit loved.
```

File: wiki/Community/Project_Loved/fr.md [hunk 17]
```diff
@@ -29,10 +25,10 @@ Certains critères très minimes doivent être respectés pour que les beatmaps
...
 - Son créateur n'est ni banni ni restreint
 - Au moins une difficulté peut être jouable
```
Comment (Ganondorf-1) [inline wiki/Community/Project_Loved/fr.md:27]:
```suggestion
- Au moins une difficulté peut être jouée jusqu'au bout sans échouer
```

File: wiki/Community/Project_Loved/fr.md [hunk 18]
```diff
@@ -46,7 +42,7 @@ Tout problème lié à ces règles est généralement résolu pendant que la map
 
 ### Priorité des nominations
 
-Il n'existe pas de règles strictes régissant les maps qu'un capitaine doit proposer pour loved, mais en général, les maps proposées sont très connues et appréciées par la communauté (comme l'indique le titre "loved"). Parfois, une beatmap sera également préféré s'il est nettement plus ancien que les autres ou si elle contiennent des beatmaps avec des styles de mapping uniques et de grande qualité.
+Il n'existe pas de règles strictes régissant les beatmaps qu'un capitaine doit proposer pour la catégorie loved, mais en général, les beatmaps proposées sont très connues et appréciées par la communauté (comme l'indique le titre "loved"). Parfois, une beatmap sera également préféré s'il est nettement plus ancien que les autres ou si elle contiennent des beatmaps avec des styles de mapping uniques et de grande qualité.
```
Comment (Ganondorf-1) [inline wiki/Community/Project_Loved/fr.md:45]:
```suggestion
Il n'existe pas de règles strictes régissant les beatmaps qu'un capitaine doit proposer pour la catégorie loved, mais en général, les beatmaps proposées sont très connues et appréciées par la communauté (comme l'indique le titre "loved"). Parfois, un beatmapset sera également préféré s'il est nettement plus ancien que les autres ou s'il contient des difficultés avec des styles de mapping uniques et de haute qualité.
```

File: wiki/Community/Video_series/fr.md [hunk 19]
```diff
@@ -0,0 +1,7 @@
+# Série de vidéos communautaires
```
Comment (Ganondorf-1) [inline wiki/Community/Video_series/fr.md:1]:
```suggestion
# Séries vidéo communautaires
```

File: wiki/Community/fr.md [hunk 20]
```diff
@@ -0,0 +1,32 @@
...
+
+- [Community mentorship program](/wiki/Community/Community_Mentorship_Program)
+- [Mappers' Guild](/wiki/Community/Mappers_Guild)
+- [osu! wiki](/wiki/osu!_wiki)
+- [Project Loved](/wiki/Community/Project_Loved)
+
+## Initiatives
+
+- [osu! community meetings](/wiki/Community/osu!_Community_Meetings)
+- [Série de vidéos](/wiki/Community/Video_series)
```
Comment (Ganondorf-1) [inline wiki/Community/fr.md:24]:
```suggestion
- [Séries vidéo](/wiki/Community/Video_series)
```

File: wiki/Community/Project_Loved/fr.md [hunk 21]
```diff
@@ -1,25 +1,21 @@
----
-no_native_review: true
----
-
 # Project Loved
 
 Le **Project Loved** est la plateforme de vote qui promeut les beatmaps dans la catégorie [loved](/wiki/Beatmap/Category#loved). Il est géré principalement par la [Project Loved Team](/wiki/People/The_Team/Project_Loved_Team).
 
 ## Déplacer une beatmap vers Loved
```
Comment (Ganondorf-1) [inline wiki/Community/Project_Loved/fr.md:5]:
```suggestion
## Déplacer une beatmap vers la catégorie loved
```

File: wiki/Community/fr.md [hunk 22]
```diff
@@ -0,0 +1,32 @@
+# Communauté
+
+Cette page répertorie les moyens par lesquels les personnes de tout le site d'osu! peuvent entrer en contact les unes avec les autres. Si vous êtes à la recherche d'opportunités de contribution, mais ne savez pas par où commencer, assurez-vous de lire [Comment vous pouvez aider !](/wiki/Community/How_you_can_help!).
```
Comment (Ganondorf-1) [inline wiki/Community/fr.md:3]:
```suggestion
Cette page répertorie les moyens par lesquels les utilisateurs de tout le site d'osu! peuvent entrer en contact les uns avec les autres. Si vous êtes à la recherche d'opportunités de contribution, mais ne savez pas par où commencer, assurez-vous de lire [Comment vous pouvez aider !](/wiki/Community/How_you_can_help!).
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

File: wiki/Community/Project_Loved/fr.md (hunk 13 above)
Comment (DoryanR) [reply wiki/Community/Project_Loved/fr.md:9]: Beatmapsets est un mot qui ne doit plus servir désormais, je vais le remplacer par `beatmap`

Comment (DoryanR) [reply wiki/Community/Project_Loved/fr.md:9]:
```suggestion
Les membres de l'équipe du Project Loved appelés **capitaines** nomment régulièrement des beatmaps pour un mode de jeu spécifique. Ces beatmaps proviennent presque toujours de propositions de la communauté d'osu!.
```

File: wiki/Community/Project_Loved/fr.md (hunk 18 above)
Comment (DoryanR) [reply wiki/Community/Project_Loved/fr.md:45]:
```suggestion
Il n'existe pas de règles strictes régissant les beatmaps qu'un capitaine doit proposer pour la catégorie loved, mais en général, les beatmaps proposées sont très connues et appréciées par la communauté (comme l'indique le titre "loved"). Parfois, une beatmap sera également préféré s'il est nettement plus ancien que les autres ou s'il contient des difficultés avec des styles de mapping uniques et de haute qualité.
```

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**